src/test/iterative_sequence/CMakeLists.txt

```diff
@@ -26,32 +26,32 @@
 
 cmake_minimum_required(VERSION 3.17)
 
-project(tritongenerativesequencebackend LANGUAGES C CXX)
+project(tritoniterativesequencebackend LANGUAGES C CXX)
 
 #
-# libtriton_generative_sequence.so
+# libtriton_iterative_sequence.so
 # Shared library implementing the Triton Sequence Backend API
 #
-configure_file(src/libtriton_generative_sequence.ldscript libtriton_generative_sequence.ldscript COPYONLY)
+configure_file(src/libtriton_iterative_sequence.ldscript libtriton_iterative_sequence.ldscript COPYONLY)
 
 add_library(
-  triton-generative-sequence-backend SHARED
-  src/generative_sequence.cc
+  triton-iterative-sequence-backend SHARED
+  src/iterative_sequence.cc
 )
 
 add_library(
-  TritonGenerativeSequenceBackend::triton-generative-sequence-backend ALIAS triton-generative-sequence-backend
+  TritonIterativeSequenceBackend::triton-iterative-sequence-backend ALIAS triton-iterative-sequence-backend
```

Comment:
I think changes are missing in `TritonGenerativeSequenceBackendConfig.cmake.in` both filename and its contents
![image](https://github.com/triton-inference-server/server/assets/21284872/ffdcad07-99d0-4700-a2e9-a3e7987938b7)